.azure-pipelines/analyze-and-test-template.yml

```diff
@@ -7,7 +7,9 @@ jobs:
   - job:
     displayName: ${{ format('CrashReporter {0} Analyze and Test', platform) }}
     pool:
-      vmImage: internal-macos-10.15
+      vmImage: macOS-12
+    variables:
+      XCODE_PATH: '/Applications/Xcode_14.0.1.app/Contents/Developer'
```

Comment:
But I can see we were doing it in the past. Maybe field for future improvement :)

Reply:
The idea is to use a specific version of Xcode. If we don't provide this path, it will use a default one which may change on CI agent update. Currently default is xcode 13. We can use it for testing as well.